R/g.report.part4.R

```diff
@@ -194,13 +194,13 @@ g.report.part4 = function(datadir = c(), metadatadir = c(), loglocation = c(),
           if (length(data_cleaning_file) > 0) {
             DaCleanFile = data.table::fread(data_cleaning_file, data.table = FALSE)
             if ("night_part4" %in% colnames(DaCleanFile)) {
-              days2exclude = which(nightsummary$ID %in% DaCleanFile$ID & nightsummary$night %in% DaCleanFile$night_part4)
-              if (length(days2exclude) == 0 & inherits(x = nightsummary$ID, "character")) {
-                # Try again by now attempt to read ID as numeric
-                options(warn = -1)
-                days2exclude = which(as.numeric(nightsummary$ID) %in% DaCleanFile$ID & nightsummary$night %in% DaCleanFile$night_part4)
-                options(warn = 0)
-              }
+              days2exclude = which(paste(nightsummary$ID, nightsummary$night) %in% paste(DaCleanFile$ID, DaCleanFile$night_part4))
+              # if (length(days2exclude) == 0 & inherits(x = nightsummary$ID, "character")) {
```

Comment:
Please remove old code instead of commenting it out. It will be in the git history so no need to keep a copy as a comment.